### PR TITLE
Added spring option

### PR DESCRIPTION
**Pull request by ParkGwangBeom**
Added options for spring animation.

### CODE REVIEW

File: ViewAnimator/Classes/ViewAnimator.swift [hunk 1]
```diff
@@ -39,7 +43,12 @@ public extension UIView {
 
         alpha = initialAlpha
         
-        UIView.animate(withDuration: duration, delay: delay, options: options, animations: { [weak self] in
+        UIView.animate(withDuration: duration,
+                       delay: delay,
+                       usingSpringWithDamping: dampingRatio,
+                       initialSpringVelocity: velocity,
+                       options: [.beginFromCurrentState],
```
Comment (marcosgriselli) [inline ViewAnimator/Classes/ViewAnimator.swift]: This will replace any `option` values passed

**Comment by marcosgriselli** [comment]
Thanks for the PR @ParkGwangBeom! This is a great feature to have. Would you mind adding a small description on the readme and a GIF of a spring animation?

**Comment by ParkGwangBeom** [comment]
@marcosgriselli Added to reame. Thank you :)

**Comment by ParkGwangBeom** [comment]
@marcosgriselli Do you need to resolve the issues mentioned in the code bit?

**Comment by marcosgriselli** [comment]
Merged! Thanks for the PR!

**Pull request merged**